mkosi/resources/man/mkosi.1.md

```diff
@@ -204,80 +204,67 @@ The following command line verbs are known:
 
 `--debug-shell`
-:   When executing a command in the image fails, mkosi will start an interactive
-    shell in the image allowing further debugging.
+:   When executing a command that fails to build an image, mkosi will start an interactive shell inside of the
+    interrupted or failed image build. This feature helps to acquire additional insights for troubleshooting.
 
 `--debug-workspace`
-:   When an error occurs, the workspace directory will not be deleted.
+:   When an error occurs, the workspace directory will not be deleted. The workspace files that existed
+    before an interruption, can be verified.
 
 `--debug-sandbox`
 :   Run `mkosi-sandbox` with `strace`.
 
 `--version`
-:   Show package version.
-
-`--help`, `-h`
-:   Show brief usage information.
+:   Show package version, build from source to have the latest version ahead of a new release.
 
 `--genkey-common-name=`
-:   Common name to be used when generating keys via mkosi's `genkey` command. Defaults to `mkosi of %u`, where
-    `%u` expands to the username of the user invoking mkosi.
+:   Common name to be used when generating keys via mkosi's `genkey` command. Defaults to `mkosi of %u`,
+    where `%u` expands to the username of the user invoking mkosi.
 
 `--genkey-valid-days=`
 :   Number of days that the keys should remain valid when generating keys via mkosi's `genkey` command.
     Defaults to two years (730 days).
 
 `--auto-bump=`, `-B`
-:   If specified, after each successful build the version is bumped in a
-    fashion equivalent to the `bump` verb, in preparation for the next
-    build. This is useful for simple, linear version management: each
-    build in a series will have a version number one higher then the
-    previous one.
+:   Automatically increment, or bump, the version know to `mkosi`, after a successful build. Each build in a
+    series will have a version number one higher then the previous one. See also the description of the
+    related `bump` verb above.
 
 `--doc-format`
-:   The format to show the documentation in. Supports the values `markdown`,
-    `man`, `pandoc`, `system` and `auto`. In the case of `markdown` the
-    documentation is shown in the original Markdown format. `man` shows the
-    documentation in man page format, if it is available. `pandoc` will generate
-    the man page format on the fly, if `pandoc` is available. `system` will show
-    the system-wide man page for mkosi, which may or may not correspond to the
-    version you are using, depending on how you installed mkosi. `auto`, which is
-    the default, will try all methods in the order `man`, `pandoc`, `markdown`,
-    `system`.
+:   The format to view documentation. The default, `auto` will try the 4 supported methods in this order:
+    `man`, `pandoc`,`markdown` and `system`.  This is done in an effort to avoid unsupported formats on the
+    host. `system` will show the system-wide man page, `pandoc` can generate the man page format on the fly.
```

Comment:
That `auto` tries all of them is less important that describing all options and this loses the descriptions of `man` and `markdown`, also `system` has lost the somewhat important information, that while the other options come from something that this particular installation of mkosi controls, no matter how it arrived on your disk, the last one might not and therefore might not reflect reality. The previous version could have done with a few paragraphs, though.